Dockerfile

```diff
@@ -8,10 +8,12 @@ RUN go mod download
 
 COPY . .
```

Comment:
This copies all the files, and later you specificaly copy scripts which does not make sense, if you want to explicitly copy scripts then exclude them from this copy command, otherwise remove additional copying of scripts. IMHO it is better to exclude them from this copy command